### PR TITLE
chore: support .env template at root

### PR DESCRIPTION
**Pull request by castastrophe**
## Description

Another focused extraction from the build rework - the utility that copies the .env settings from a user's root to the project to ease the following workflow:

- git clean -dfx
- yarn install

Checking out a clean repo is a critical part of the workflow but in doing so, it removes any uncommitted utilities such as the .env file with the Chromatic project token. Any time a yarn install is run, this utility script tries to copy variables listed in the example file from root.

## How and where has this been tested?

- touch ~/.env
- echo "CHROMATIC_PROJECT_TOKEN=foo-bar" >> ~/.env
- git clean -dfx
- yarn install

Expect to see the following message: `🪄 Using .env.example to generate a local .env`

When opening up `.env` at the project root, expect to see `CHROMATIC_PROJECT_TOKEN=foo-bar` at the top of the file.

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [x] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

**Comment by pfulton** [comment]
Tested it out locally and works as expected! Ship it.

File: components/pagination/metadata/mods.md [hunk 1]
```diff
@@ -6,5 +6,3 @@
 | `--mod-pagination-counter-margin-inline-start` |
 | `--mod-pagination-page-button-inline-spacing`  |
 | `--mod-pagination-textfield-width`             |
-| `--mod-textfield-min-width`                    |
```
Comment (pfulton) [inline components/pagination/metadata/mods.md:9]: Did you mean for this to go in as well? If so, could you separate out this change to its own commit? 😬

**Review by pfulton — APPROVED** [review]
I'm good with this, but I did leave one comment about the mods file.

File: components/pagination/metadata/mods.md (hunk 1 above)
Comment (castastrophe) [reply components/pagination/metadata/mods.md:9]: I can pull it out - it's a side-effect of the updated mods script going in. Probably pagination's PR was merged after the mods script was opened for code review.
